src/commands/sites/sites-list.ts

```diff
@@ -8,16 +8,14 @@ import BaseCommand from '../base-command.js'
 
 export const sitesList = async (options: OptionValues, command: BaseCommand) => {
   const { api } = command.netlify
-  /** @type {import('ora').Ora} */
   let spinner
   if (!options.json) {
     spinner = startSpinner({ text: 'Loading your sites' })
   }
   await command.authenticate()
 
   const sites = await listSites({ api, options: { filter: 'all' } })
-  if (!options.json) {
-    // @ts-expect-error TS(2345) FIXME: Argument of type '{ spinner: Ora | undefined; }' i... Remove this comment to see the full error message
+  if (spinner) {
```

Comment:
These are equivalent, but this is less leaky and makes TS happy. We only define a spinner (upstream) when `--json` isn't true.